portfolio/src/main/java/com/google/sps/servlets/RandomStringServlet.java

```diff
@@ -0,0 +1,18 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/tester")
```

Comment:
Change this to a more descriptive name, such as "fetch-hello-world-string".

Reply:
Just did so in the last commit